src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -443,7 +443,7 @@ public void changeCandidateCode(CellEditEvent cellEditEvent) {
   @Override
   public void initialize(URL location, ResourceBundle resources) {
     Logger.addGuiLogging(this.textAreaStatus);
-    Logger.log(Level.INFO, "Opening tabulator GUI...");
+    Logger.log(Level.INFO, String.format("Welcome to the %s!", Main.APP_NAME));
```

Comment:
Fine to keep this line, but I'd still like to keep mention of opening the GUI here. That way, when we're parsing the logs, we can tell if they used GUI vs. CLI, e.g.:

`Logger.log(Level.INFO, "Opening tabulator GUI...");`